Dockerfile-pdfium.demo

```diff
@@ -0,0 +1,41 @@
+FROM trailofbits/polytracker
+MAINTAINER Carson Harmon <carson.harmon@trailofbits.com>
+
+WORKDIR /polytracker/the_klondike
+
+RUN apt-get install -y git pkg-config
+
+WORKDIR /
+
+RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+ENV PATH=$PATH:/depot_tools/
+WORKDIR /polytracker/the_klondike/
+
+RUN gclient config --unmanaged https://pdfium.googlesource.com/pdfium.git
+RUN gclient sync
+
+RUN apt update
+RUN apt install software-properties-common -y
+RUN add-apt-repository "deb http://archive.canonical.com/ $(lsb_release -sc) partner"
+RUN apt update
+RUN apt install sudo -y
+RUN apt install vim -y
+#For some reason this isnt in our llvm-7 install?
```

Comment:
I think we need to add `apt-get install lld-7` in the base `trailofbits/polytracker` image.